app/config/config.yml

```diff
@@ -15,6 +15,8 @@ parameters:
     env(PS_LOG_OUTPUT): "%kernel.logs_dir%/%kernel.environment%.log"
     mail_themes_uri: "/mails/themes"
     mail_themes_dir: "%kernel.project_dir%%mail_themes_uri%"
+    ps_tmp_img_dir: "/var/www/PrestaShop/img/tmp/"
```

Comment:
Why?? :thinking:

Reply:
To be able to inject other directory from config_test.yml by overriding services. Any better ideas?

Reply:
relatave path needed here ofcourse :)